apps/indexer-coordinator/src/payg/payg.sync.service.ts

```diff
@@ -38,79 +39,63 @@ export class PaygSyncService implements OnApplicationBootstrap {
 
   @Cron(CronExpression.EVERY_MINUTE)
   async syncStateChannelsPeriodically() {
+    if (this.syncingStateChannels) {
+      logger.debug(`Bypass syncing state channels...`);
+      return;
+    }
+    this.syncingStateChannels = true;
     try {
-      logger.debug(`Load from Subquery Project...`);
+      logger.debug(`Syncing state channels from Subquery Project...`);
       const hostIndexer = await this.account.getIndexer();
       if (!hostIndexer) {
         logger.debug(`Indexer not found, will sync state channel later...`);
+        this.syncingStateChannels = false;
         return;
       }
-      const channels = await this.paygQueryService.getStateChannels(hostIndexer);
+      const stateChannels = await this.paygQueryService.getStateChannels(hostIndexer);
+      const localAliveChannels = await this.paygService.getAliveChannels();
+
+      const stateChannelIds = stateChannels.map((stateChannel) =>
```

Comment:
how about channels of status `TERMINATED`, but is `OPEN` in local db? Should call syncChannel for them as well